azure-pipelines.yml

```diff
@@ -227,7 +227,7 @@ stages:
             TestResultsArtifactName: 'DotnetTestResultsNet48'
             ProjectFilePath: 'tests\SonarAnalyzer.Test\SonarAnalyzer.Test.csproj'
           Net70:
-            FrameworkMoniker: 'net7.0'
+            FrameworkMoniker: 'net7.0-windows'
```

Comment:
what extra does targetting OS specific-tfm offer here? (I honestly don't understand),.

Reply:
It doesn't work without it.
`SonarAnalyzer.TestFramework` has a framework reference to `Microsoft.WindowsDesktop.App.WindowsForms`. This was copied from the old test project.
Unfortunatelly, everything that references TestFramework now needs to have `-windows` suffix due to this. It wouldn't compile otherwise.

I will get rid of this problem completely later (not sure if in this sprint).